.github/workflows/ci.yml

```diff
@@ -26,10 +26,19 @@ jobs:
     name: Tests
     timeout-minutes: 5
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # how to say "not these" so we don't miss anything?
+        # waiting on a an api from vitest for querying
+        # the list of tests ahead of time before running them.
+        slow-test:
+          - defaults
+          - addon-location
+          - test-app-location
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/pnpm
```

Comment:
Shouldn't we setup node explicitly here? To
* run with our minimum node version (14)
* to allow use of yarn caching 

Like 
```yaml
- uses: actions/setup-node@v3
  with:
    node-version: 14
    cache: yarn
```

This would speed up our `install` call in tests, wouldn't it?

Reply:
The actions folder abstraction sets all this here:

https://github.com/embroider-build/addon-blueprint/blob/main/.github/actions/pnpm/action.yml#L6

Currently uses volta for node version, so we'd want to set our volta node to minimum supported node

Reply:
Ah ok, got it. Though it still seems the yarn cache is not persisted across CI runs, which is used by the tests itself, right?

Not a huge deal though, we can optimize this later if we see CI times getting out of bounds...

Reply:
yeah -- I'll PR something next related to this